Doc/library/tkinter.rst

```diff
@@ -67,13 +67,33 @@ Or, more often::
    from tkinter import *
 
 
-.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1)
-
-   The :class:`Tk` class is instantiated without arguments. This creates a toplevel
-   widget of Tk which usually is the main window of an application. Each instance
-   has its own associated Tcl interpreter.
-
-   .. FIXME: The following keyword arguments are currently recognized:
+.. class:: Tk(screenName=None, baseName=None, className='Tk', useTk=1, sync=0, use=None)
+
+   Return a toplevel Tk widget, which is usually the main window of an
+   application.  Each instance has its own associated Tcl interpreter.
+
+   The :class:`Tk` class is typically instantiated using all default values.
+   However, the following keyword arguments are currently recognized:
+
+   *screenName*
+      Sets the DISPLAY environmental variable. (X11 only)
+   *baseName*
+      Reads the profile file :file:`baseName.tcl` into the Tcl interpreter and
+      calls ``exec`` on the contents of :file:`baseName.py`.
+   *className*
+      Name of the widget class.  Used as a profile file name
+      (:file:`className.tcl` and :file:`className.py` load before the
+      *baseName* files), and also used as the name with which Tcl is
+      invoked (*argv0* in *interp*).
+   *useTk*
+      If True, initialize the Tk subsystem.  :func:`Tcl` sets this to False.
+   *sync*
+      If True, execute all X server commands synchronously, so that errors are
+      reported immediately.
+   *use*
+      Specifies that the main window for the application is to be embedded in
+      the window whose identifier is given, instead of being created as an
+      independent toplevel window.
```

Comment:
What sort of window can be a host -- X11 only? Tk?, and what sort of identifier can be given (same questions).

Reply:
This should work on all three platforms. The identifier of Tk window is returned by `winfo_id()`, but the window can be embedded not only in a Tk window. The `Toplevel` widget takes the same *use* argument.

You can create a GUI application, pass an identifier of an inner window to a subproces, and the output of this subprocess be rendered in the bounds of this window instead of a separate toplevel window.

Reply:
I copied that from the manpage for wish  (https://www.tcl.tk/man/tcl/UserCmd/wish.htm).  Maybe I should copy more of the text?